package-lock.json

```diff
@@ -0,0 +1,14664 @@
+{
```

Comment:
Non blocking but the project is built with yarn. I'm not excited about having 2 lock files to juggle in the repo. I don't think it's a problem for local dev.